packages/core/src/button/Button.css

```diff
@@ -132,3 +84,93 @@
   cursor: var(--saltButton-cursor-disabled, var(--salt-actionable-cursor-disabled));
   border-color: var(--saltButton-borderColor-disabled, var(--button-borderColor-disabled));
 }
+
+.saltButton-solid-accent {
+  --button-text-color: var(--salt-actionable-accent-bold-foreground);
+  --button-text-color-hover: var(--salt-actionable-accent-bold-foreground-hover);
+  --button-text-color-active: var(--salt-actionable-accent-bold-foreground-active);
+  --button-text-color-disabled: var(--salt-actionable-accent-bold-foreground-disabled);
+  --button-background: var(--salt-actionable-accent-bold-background);
+  --button-background-active: var(--salt-actionable-accent-bold-background-active);
+  --button-background-disabled: var(--salt-actionable-accent-bold-background-disabled);
+  --button-background-hover: var(--salt-actionable-accent-bold-background-hover);
+  --button-borderColor: var(--salt-actionable-accent-bold-border);
+  --button-borderColor-hover: var(--salt-actionable-accent-bold-border-hover);
+  --button-borderColor-active: var(--salt-actionable-accent-bold-border-active);
+  --button-borderColor-disabled: var(--salt-actionable-accent-bold-border-disabled);
+}
+
+.saltButton-outline-accent {
+  --button-text-color: var(--salt-actionable-accent-foreground);
+  --button-text-color-hover: var(--salt-actionable-accent-foreground-hover);
+  --button-text-color-active: var(--salt-actionable-accent-foreground-active);
+  --button-text-color-disabled: var(--salt-actionable-accent-foreground-disabled);
+  --button-background: var(--salt-actionable-accent-background);
+  --button-background-hover: var(--salt-actionable-accent-background-hover);
+  --button-background-active: var(--salt-actionable-accent-background-active);
+  --button-background-disabled: var(--salt-actionable-accent-background-disabled);
+  --button-borderColor: var(--salt-actionable-accent-border);
+  --button-borderColor-hover: var(--salt-actionable-subtle-border-hover);
```

Comment:
`--button-borderColor-hover` for `outline-accent` should point to `--salt-actionable-accent-border-hover` instead of `--salt-actionable-subtle-border-hover`

Reply:
corrected